myna/application/additivefoam/additivefoam.py

```diff
@@ -126,3 +127,19 @@ def copy(self, case_dir, mesh_path, mesh_dict):
                     yaml.dump(mesh_dict, f, default_flow_style=None)
 
         return use_existing_mesh
+
+    def update_material_properties(self, case_dir):
+
+        material = self.settings["data"]["build"]["build_data"]["material"]["value"]
+        material_data = os.path.join(
+            os.environ["MYNA_INSTALL_PATH"],
+            "resources",
+            "mist_material_data",
+            f"{material}.json",
+        )
+        mat = mist.core.MaterialInformation(material_data)
+        transport_filepath = os.path.join(case_dir, "constant", "transportProperties")
+        thermo_filepath = os.path.join(case_dir, "constant", "thermoPath")
+        mat.write_additivefoam_input(
+            transport_file=transport_filepath, thermo_file=thermo_filepath
```

Comment:
Technically not needed here since these are passed in the expected order. But in general, you can pass optional args (i.e., args with default values) in any order as long as you pass them as named args, so I find it as a handy visual indication that default parameters of a function are being overwritten.